### PR TITLE
[Docs] [Quick start] Build a python application page for ysql-django

### LINKED ISSUE #11027: [Docs] Document Django in Quick start.
### Description

Document provided by @Sfurti-yb : https://docs.google.com/document/d/1yVtkHS5FAd7oNOnjC95DpjOTK5LykH6p7kWEyRp4MGo/edit

### PR DESCRIPTION
**Pull request by aishwarya24**
Resolves #11027, #11067 

- Verified the steps and included a new build a python application page in latest and stable.
- Fixed broken links in Quick start and Build apps for versions 2.6, 2.4 and 2.2. 
   
   https://docs.yugabyte.com/v2.6/quick-start/install/
   https://docs.yugabyte.com/v2.6/quick-start/create-local-cluster/
   https://docs.yugabyte.com/v2.6/quick-start/explore/ 
   https://docs.yugabyte.com/v2.4/quick-start/build-apps/java/ (changed all the other languages as well)
  (Changed for 2.4 and 2.2 as well) 
  

@netlify /latest//quick-start/build-apps/python/ysql-django/

### CODE REVIEW

File: docs/content/latest/quick-start/build-apps/python/ysql-django.md [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+}
+```
+
+- Generate a Django secret key using any django secret key generator and paste the generated key in the following line of the `settings.py` file:
+
+```python
+SECRET_KEY = 'YOUR-SECRET-KEY'
+```
+
+- Create a database using the `YugabyteDB SQL shell`(ysqlsh) by navigating to the location of your local [YugabyteDB](#yugabytedb) cluster created above.
```
Comment (polarweasel) [inline docs/content/latest/quick-start/build-apps/python/ysql-django.md:106]:
```suggestion
- Create a database using the YugabyteDB YSQL shell (ysqlsh). From the location of your local [YugabyteDB](#yugabytedb) cluster, run the following shell command:
```

File: docs/content/latest/quick-start/build-apps/python/ysql-django.md [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+
+```sh
+bin/ysqlsh -c "CREATE DATABASE ysql_django"
+```
+
+- From the `orm-examples/python/django` directory, create the migrations, and migrate the changes to the database using the following command:
```
Comment (polarweasel) [inline docs/content/latest/quick-start/build-apps/python/ysql-django.md:112]:
```suggestion
- From the `orm-examples/python/django` directory, run the following command to create the migrations and migrate the changes to the database:
```

**Review by polarweasel — CHANGES_REQUESTED** [review]
This is looking really good. Just a few minor changes (and one that points to some changes required in the orm-examples repo, sorry!).

File: docs/content/latest/quick-start/build-apps/python/ysql-django.md [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+        'NAME': 'ysql_django',
+        'USER': 'postgres',
+        'PASSWORD': '',
+        'HOST': '127.0.0.1',
+        'PORT': '5433',
+    }
+}
+```
+
+- Generate a Django secret key using any django secret key generator and paste the generated key in the following line of the `settings.py` file:
```
[the remark this answers is not in the document]
Comment (aishwarya24) [reply docs/content/latest/quick-start/build-apps/python/ysql-django.md:100]: Thank you ! I wasn't comfortable about this either. Will make the change.

File: docs/content/latest/quick-start/build-apps/python/ysql-django.md [hunk 4]
```diff
@@ -0,0 +1,308 @@
...
+[Python 3](https://www.python.org/downloads/) or later is installed.
+
+### Django
+
+[Django 2.2](https://www.djangoproject.com/download/) or later is installed.
+
+## Clone the orm-examples repository
+
+```sh
+$ git clone https://github.com/yugabyte/orm-examples.git
```
[the remark this answers is not in the document]
Comment (aishwarya24) [reply docs/content/latest/quick-start/build-apps/python/ysql-django.md:71]: Understood and noted. Thanks.

File: docs/content/latest/quick-start/build-apps/python/ysql-django.md [hunk 5]
```diff
@@ -0,0 +1,308 @@
...
+```
+
+The REST API server will start and listen for your requests at `http://localhost:8080`.
+
+## Send requests to the application
+
+Create 2 users.
+
+```sh
+$ curl --data '{ "firstName" : "John", "lastName" : "Smith", "email" : "jsmith@yb.com" }' \
```
[the remark this answers is not in the document]
Comment (aishwarya24) [reply docs/content/latest/quick-start/build-apps/python/ysql-django.md:133]: This also requires us to change the domains in around 10 other quick start pages when I search for `orm-examples` in latest and stable alone. Shall I create a separate issue to fix those pages ? @polarweasel

File: docs/content/latest/quick-start/build-apps/python/ysql-django.md [hunk 6]
```diff
@@ -0,0 +1,308 @@
...
+## Set up the application
+
+- Customize the database connection setting according to your environment in `ybstore/settings.py` under `orm-examples/python/django` directory:
+
+```python
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'ysql_django',
+        'USER': 'postgres',
```
[the remark this answers is not in the document]
Comment (aishwarya24) [reply docs/content/latest/quick-start/build-apps/python/ysql-django.md:92]: @Sfurti-yb , could you please confirm with this one. Pointed this question in the google doc as well.

**Review by polarweasel — APPROVED** [review]
LGTM, pending Sfurti's confirmation on the username...

**Pull request merged**